### ISSUE TITLE
en/fr language mixup

### ISSUE THREAD

**Issue by Dmole** [frappe/books#624]
![french_q](https://github.com/frappe/books/assets/1793802/46f29c0e-2641-4f4b-8ddb-cd4eab661540)

**Comment by 18alantom** [comment]
This is cause the [French Chart of Accounts](https://github.com/frappe/books/blob/master/fixtures/verified/fr.json) is in french, and so all the account names are in French.

This is auto-set in the Setup Wizard when selecting the country.

**Comment by Dmole** [comment]
Setup > Settings > Locale == en-CA

( not fr-CA )

Does FBooks only support 1/13 of Canadian provinces?

**Comment by Dmole** [comment]
Maybe FBooks could use the localeplanet library to improve localization;

https://github.com/Khan/localeplanet

https://www.localeplanet.com/support/howto-localize-js.html

**Comment by TimRyanKaslovia** [comment]
I assume this is because there is a French user who supplied the data template. I am installing with the default English CofA and will provide a corrected EN chart when completed if Alan can reload the Chart feed for Canada.

**Comment by bryanparadis** [comment]
This is still an issue with the current release. You pick Canada as the locale and it automatically uses the French Canadian Chart of Accounts. I instead chose Standard Chart of Accounts afterwards.

**Comment by Gadha2311** [comment]
Could you please confirm if you are still encountering this issue, or has it been resolved on your end?

**Comment by Dmole** [comment]
Yes it is still a problem because this software still does not distinguish between country and language and so is useless to some.

**Comment by Gadha2311** [comment]
when creating a new company or during the initial setup, you can select the country directly.

**Comment by Dmole** [comment]
and get the wrong language.

**Comment by bryanparadis** [comment]
> when creating a new company or during the initial setup, you can select the country directly.

Yes. You select Canada and get French by default. This is the issue. Canada has both languages with English being the dominant one but French being the default in the application

**Comment by Gadha2311** [comment]
You can change the language when switching the database.

**Comment by Dmole** [comment]
@Gadha2311

Either you don't know what you are talking about or I don't understand it.
This is what users see:

1. Welcome screen has ***English*** selected by default.
2. 2nd screen (create company) ***Canada*** is selected by the user.
3. 3rd screen (get started) seems fine
4. try to do anything like Chart of accounts and it's ***French*** for no good reason (a ***bug***).
5. change database just loops back to # 1
6. User uninstalls unusable software.


The user selecting the combination of English + Canada should result in ***en-CA.json*** being used ( not ***fr.json*** ).
The correct way to fix this is to implement localization properly instead of whatever chaotic thing is going on now.

1. Rename json files to $2LetterLanguage-$2LetterCountry.json
2. Make the  Country selector limited by the Language choice (based on what json files exist).

**Comment by bryanparadis** [comment]
> You can change the language when switching the database.

I think we've both described it clearly enough.

**Comment by bryanparadis** [comment]
The current default for Canada is the French chart of accounts
<img width="896" height="294" alt="Image" src="https://github.com/user-attachments/assets/d699bdea-9978-41f5-ba9d-7a9c059223e5" />

Expected default Chart of Accounts when selecting Canada:
<img width="1012" height="319" alt="Image" src="https://github.com/user-attachments/assets/869c3429-d0d0-4c2e-af15-a46193a6f366" />

**Comment by bryanparadis** [comment]
This is set here

https://github.com/frappe/books/blob/2f028f42be70a1dbd45bd4c6483d57df2fc64805/models/baseModels/SetupWizard/SetupWizard.ts#L28-L29
